organization/components/turfs/TurfGrid.tsx

```
@@ -0,0 +1,68 @@
+    );
+  }
+
+  if (turfs.length === 0) {
+    return (
+      <div className="flex flex-col items-center justify-center py-16 text-gray-500">
+        <svg
+          className="w-16 h-16 mb-4 text-gray-400"
+          fill="none"
+          stroke="currentColor"
+          viewBox="0 0 24 24"
+        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)